packages/schema-poller/src/schemaPoller.ts

```diff
@@ -1,32 +1,23 @@
-import neo4j, { Config } from 'neo4j-driver';
+import { EventEmitter } from 'events';
+import { Config } from 'neo4j-driver';
+import { initializeDriver } from './driverInitializer';
 import { MetadataPoller } from './metadataPoller';
 import { Neo4jConnection } from './neo4jConnection';
 import { listDatabases } from './queries/databases.js';
 
 export class Neo4jSchemaPoller {
   public connection?: Neo4jConnection;
   public metadata?: MetadataPoller;
+  public connectionEvents = new EventEmitter();
```

Comment:
create an EventEmitter that we will use for communication between the schema poller and the language server